src/containers/flows/BuyTokenRedirect/BuyTokenRedirect.module.scss

```diff
@@ -0,0 +1,160 @@
+.WildPriceContainer {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	position: relative;
+	background: none;
+	height: 100%;
+	padding: 4px 16px;
+	cursor: pointer;
+	transition: all 0.2s ease-in-out 0s;
+
+	&::after {
```

Comment:
I noticed that the `::after` gradient overlay is above the text which changes the white colour, can we set the z-index so the gradient is behind the text and doesn't change the white point?

(colour values below are the main text colours the user sees in current version - both should be #fff)

![Screen Shot 2022-01-14 at 4 59 32 PM](https://user-images.githubusercontent.com/12437916/149449283-5974c5c0-4a0d-46f2-acde-8c56f33c696b.png)